### PR TITLE
fix(GeoSearch): override button style

### PR DESCRIPTION
**Pull request by samouss**
**Before**

screen shot 2018-03-05 at 20 10 06

**After**

screen shot 2018-03-05 at 20 10 26

### CODE REVIEW

File: src/css/theme/_geo-search.scss [hunk 1]
```diff
@@ -43,6 +46,7 @@
   border: solid 1px #D4D8E3;
   border-radius: $border-radius;
   padding: 8px 15px;
+  outline: none;
```
Comment (Haroenv) [inline src/css/theme/_geo-search.scss]: I don't think we should remove the outline without replacement though, see: [outlinenone.com](http://www.outlinenone.com)

Comment (bobylito) [reply src/css/theme/_geo-search.scss]: Not false. What alternative can you propose @Haroenv ?

Comment (samouss) [reply src/css/theme/_geo-search.scss]: Completely agree! It's just a bad habits 😄

Comment (Haroenv) [reply src/css/theme/_geo-search.scss]: 1. don't change outline
2. we should have a proper outline style which isn't ugly in Chrome; cc @ronanlevesque

**Pull request merged**